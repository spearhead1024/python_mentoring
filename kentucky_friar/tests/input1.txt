So I was fixing to ask him, "Don't you want to go fishing?" I was dying, says I,
to go for a swing, maybe do some swimming, too.
