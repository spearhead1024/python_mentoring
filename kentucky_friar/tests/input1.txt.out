So I was fixin' to ask him, "Do y'all want to go fishin'?" I was dyin'
to go for a swing and maybe do some swimmin', too.
